ui/components/app/terms-of-use-popup/terms-of-use-popup.js

```diff
@@ -0,0 +1,1155 @@
+            incorporated into this Agreement, each as may be updated by us from
+            time to time.
+          </Text>
+          <Text variant={TextVariant.bodySm} marginBottom={4}>
+            “Privacy Policy” means the privacy policy located at{' '}
+            <a
+              href="https://consensys.net/privacy-policy"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              https://consensys.net/privacy-policy
+            </a>{' '}
+            (and any successor or related locations designated by us), as it may
+            be updated by us from time to time.
+          </Text>
+          <Text variant={TextVariant.bodySm} marginBottom={4}>
+            “Service Offerings” means the Services (including associated APIs),
+            Our Content, Our Marks, and any other product or service provided by
+            us under this Agreement. Service Offerings do not include
+            Third-Party Content or Third-Party Services.
+          </Text>
+          <Text variant={TextVariant.bodySm} marginBottom={4}>
+            “Suggestions” means all suggested improvements to the Service
+            Offerings that you provide to us..
+          </Text>
+          <Text variant={TextVariant.bodySm} marginBottom={4}>
+            “Term” means the term of this Agreement described in Section 6.1.
+          </Text>
+          <Text variant={TextVariant.bodySm} marginBottom={4}>
+            “Termination Date” means the effective date of termination provided
+            in accordance with Section 6, in a notice from one party to the
+            other.
+          </Text>
+          <Text variant={TextVariant.bodySm} marginBottom={4}>
+            “Third-Party Content” means Content made available to you by any
+            third party on the Site or in conjunction with the Offerings.
+          </Text>
+          <Text variant={TextVariant.bodySm} marginBottom={4}>
+            “Your Content” means content that you or any End User transfers to
+            us, storage or hosting by the Offerings in connection with account
+            and any computational results that you or any End User derive from
+            the foregoing through their use of the Offerings, excluding however
+            any information submitted to a blockchain protocol for
+            processing.&nbsp;
+          </Text>
+          <Box
+            flexDirection={FLEX_DIRECTION.ROW}
+            alignItems={AlignItems.flexStart}
+            marginLeft={3}
+            marginRight={3}
+            gap={2}
+          >
+            <CheckBox
+              id="terms-of-use__checkbox"
+              className="terms-of-use__checkbox"
+              dataTestId="terms-of-use-checkbox"
+              checked={isTermsOfUseChecked}
+              onClick={() => {
+                setIsTermsOfUseChecked(!isTermsOfUseChecked);
+              }}
+            />
+            <label
```

Comment:
Use the label component from the component-library. It will be helpful with css as well